breakwater/src/statistics.rs

```diff
@@ -221,12 +221,13 @@ impl Statistics {
         let elapsed_ms = max(1, elapsed.as_millis()) as u64;
         let frame = self.frame;
         let connections = self.connections_for_ip.values().sum();
-        let ips = self.connections_for_ip.len() as u32;
-        let legacy_ips = self
-            .connections_for_ip
-            .keys()
-            .filter(|ip| ip.is_ipv4())
-            .count() as u32;
+        let [ips, legacy_ips] =
```

Comment:
If we do this can you please rename all variables (also a struct level and other files) to ips_v4 and ips_v6?
This is to avoid the confusion that lead to this PR in the first time :see_no_evil: